source/_components/homekit.markdown

```diff
@@ -95,6 +102,11 @@ homekit:
                 required: false
                 type: string
                 default: ''
+              mode:
+                description: Operation mode of switches within HomeKit. Valid modes are `on_off`, `play_pause`, `play_stop`, and `toggle_mute`. The entity will ignore any `mode` which is not supported. If not specified, all supported modes will be added. Only applicable for `media_player` entities.
+                required: false
+                type: list
+                default: `on_off`, `play_pause`, `play_stop`, `toggle_mute`
```

Comment:
The default would change to: '`<All supported modes>`'

Reply:
I agree, but I don't think a code block should be used since it's not the actual entry. I prefer just plain text.

Reply:
I wanted to use the same syntax for placeholders as above for `entity_id`. A normal `string` might be misleading.